translate/deeplx/deeplx.go

```diff
@@ -0,0 +1,99 @@
+package deeplx
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+
+	"github.com/metatube-community/metatube-sdk-go/common/fetch"
+	"github.com/metatube-community/metatube-sdk-go/translate"
+)
+
+var _ translate.Translator = (*DeepLX)(nil)
+
+type DeepLX struct {
+	APIKey  string `json:"deepl-api-key"`
+	BaseURL string `json:"base-url"`
+}
+
+func (dpl *DeepLX) Translate(q, source, target string) (result string, err error) {
```

Comment:
```suggestion
func (dplx *DeepLX) Translate(q, source, target string) (result string, err error) {
```
Better to use `dplx` here